bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/ListImpl.java

```diff
@@ -54,7 +54,7 @@
     @JsonProperty("items")
     public Collection<ListItem> getListItems() {
         if (this.listItems == null) {
-            this.listItems = super.getPages().stream()
+            this.listItems = super.getItems().stream()
```

Comment:
Using `getItems` because the `testSearchListType` test was calling both `getItems` (to check list consistency) and `getListItems` (to test JSON export) and the mock search result iterator was being exhausted by the first call. This ensures `getPages` is not called twice. Alternatively we could move the lazy loading caching in `getPages`